RELEASE_NOTES.md

```diff
@@ -2,6 +2,128 @@
 
 <!-- newest release at the top please) -->
 
+# Prod 2019/05/08
+## Ingest
+### Ingest-ui v0.9.2.rc
+* Setting of row height automatically to show all validation errors
+* Do not allow creation of project metadata thru UI
+ 
+## Upload  v4.3.1
+* Remove "list upload area" REST API endpoint
+* Added client-side checksumming (temporarily disabled).
+* Much Internal refactoring and cleanup.
+
+## Data Store
+### Version: 2019-05-08-02-45-24-staging.release
+* Improve `PUT /files/{uuid}` description
+* Clarify and fix test expectations
+* Add basic indexing for collections.
+* Fix multipart sync completion criteria
+* Simplify GitLab job definitions
+* Prod Smoke Test Query
+* Change String Comparison
+* Isolate Terraform State File Buckets
+* Remove trufflehog dependency
+* Update TF to remove DEPRECATED messages
+* Increase GS client connection pool size
+* Cache for content type like 'application/json; foo'
+* Sync daemon sets destination content-type equal to source
+* Update subscription_v2 ddb interface calls
+* Use dynamoDB methods in async_state
+* Abstract out dynamoDB functionality from subscriptions
+
+## Secondary Analysis
+### Pipeline-tools v0.49.1
+* Update metadata schema when formatting analysis results:
+    * Change process/protocol_type to type
+    * Change file_format to format
+* Update to metadata-api library release/1.0b15
+* Subscriptions queries process_type -> type, dissociation_method -> method
+
+## Data Browser
+### Version: 156f751de3b693ffa8dabcb78ee7550bcee5fe7f
+* Samples Tab
+
+## Azul
+### Version: 53f0b12c25364a8c5f527ad2119fee48df096b40
```

Comment:
```suggestion
### Version: 6a3639a97acc46cde7d98530c604f777ecee99fd
```

Sorry, for not filling out the release notes earlier. The individual commits are the same but the merge commit has a different sha.